src/plugins/visualizations/public/plugin.ts

```diff
@@ -23,8 +23,8 @@ import {
   createStartServicesGetter,
   Storage,
   withNotifyOnErrors,
-  replaceUrlHashQuery,
 } from '@kbn/kibana-utils-plugin/public';
+import { replaceUrlHashQuery } from '@kbn/kibana-utils-plugin/common';
```

Comment:
Why are we doing this change?

Reply:
because we're moving `replaceUrlHashQuery` from `public` to `common` to have the ability to use locators also server side

Reply:
Got it thanx Matthias